File: Code_Force/Div_900/Theatre_Square.java
package Code_Force.Div_900;

import java.util.Scanner;

public class Theatre_Square {

    public static void main(String[] args) {
        Scanner cin = new Scanner(System.in);

        long n = cin.nextInt();
        long m = cin.nextInt();
        long a = cin.nextInt();

        long res = ((n + a - 1) / a) * ((m + a - 1) / a);
        System.out.println(res);

        cin.close();
    }
}

File: Code_Force/Div_1100/Theatre_Square.cpp
#include <iostream>
using namespace std;

int main()
{
    int n, m, a;
    cin >> n >> m >> a;
    long long x, y;
    if (n % a != 0)
        x = n / a + 1;
    else
        x = n / a;
    if (m % a != 0)
        y = m / a + 1;
    else
        y = m / a;
    cout << x * y;
    return 0;
}